src/org/ods/usecase/JiraUseCase.groovy

```diff
@@ -266,7 +266,7 @@ class JiraUseCase {
         this.jira.updateSelectListFieldsOnIssue(releaseStatusIssueKey, [(releaseStatusIssueReleaseManagerStatusField.id): status])
 
         if (error) {
-            this.jira.appendCommentToIssue(releaseStatusIssueKey, error.message)
+            this.jira.appendCommentToIssue(releaseStatusIssueKey, "${error.message}\nSee: ${this.steps.env.RUN_DISPLAY_URL}")
```

Comment:
@georgfedermann let's add another newline to make the last line more prominent please, thanks! Have you tried this out in Jira? Does it accept the `\n`?

Reply:
I felt the same way about the additional newline.
Yes, Jira accepts the newline. Evidence is attached to the Jira issue.